src/Symfony/Component/HttpFoundation/ResponseHeaderBag.php

```diff
@@ -88,12 +88,22 @@ public function replace(array $headers = [])
     /**
      * {@inheritdoc}
      */
-    public function all()
+    public function all(/*string $key = null*/)
```

Comment:
corresponding `@param` annotation should be added

Reply:
it's coming from the inheritdoc, should I add it anyway ?

Reply:
yes, it's a must, to express that you acknowledge the fact the argument should be added and prevent triggering a deprecation